website/docs/language/resources/syntax.mdx

```diff
@@ -150,17 +133,19 @@ resource "aws_instance" "example" {
 }
 ```
 
-Custom conditions can help capture assumptions, helping future maintainers understand the configuration design and intent. They also return useful information about errors earlier and in context, helping consumers more easily diagnose issues in their configurations.
-
-Refer to [Custom Condition Checks](/terraform/language/expressions/custom-conditions#preconditions-and-postconditions) for more details.
+[Custom Condition Checks](/terraform/language/expressions/custom-conditions#preconditions-and-postconditions) 
+can help capture assumptions, helping future maintainers 
+understand the configuration design and intent. They also return useful 
+information about errors earlier and in context, helping consumers to diagnose 
+issues in their configuration.
```

Comment:
reviewer note: changed line length and moved links to the first mention of custom conditions